pkg/controllers/config/config.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+const (
+	DefaultCloudResourcePrefix = "nephe"
+	ConfigMapName              = "nephe-config"
+	ConfigMapNamespace         = "nephe-system"
+)
+
+var (
+	ConfigFile string
+)
+
+type ControllerConfig struct {
+	CloudResourcePrefix string `yaml:"CloudResourcePrefix,omitempty"`
+}
+
+func SetConfigFile(name string) {
```

Comment:
why no move ConfigFile inside ControllerConfig struct?? avoid using global variables..

Reply:
Because controller config is specifically for nephe-controller.conf. The config file contains more than one controller conf that's why we used this approach.